<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>Алгоритм Уоршелла</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Обход в глубину</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_12"/>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Алгоритм Дейкстры</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>алгоритм Форда-Фалкерсон</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
